sim/src/mechanics/intersection.rs

```diff
@@ -561,6 +570,14 @@ impl IntersectionSimState {
                 }
                 (None, None) => {}
             }
+
+            // It's unlikely, but the player might create/destroy traffic signals close together and
+            // change the uber-turns that exist. To be safe, recalculate everywhere.
+            state.uber_turn_neighbors.clear();
```

Comment:
It actually will happen a lot. Covert stop to ts and convert midblock crosswalk to degenerate w/ crosswalk.